test/integration/targets/win_scheduled_task/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+# NOTE: The win_scheduled_task module only works on Win2012+
+
+- name: Test in normal mode (only run on Win2012+)
```

Comment:
This only works for current shippable config- should use feature sniff instead, something like `raw: Get-Command New-ScheduledTask -ErrorAction SilentlyContinue; $?` with `failed_when: False` then skip the tests if the rc != 0. I guess we could use the OS version facts too...